README.md

```diff
@@ -258,6 +258,7 @@ To run without installing:
  | click on `HOME`                        | `Ctrl`+`h` \| _Middle-click_  |
  | click on `BACK`                        | `Ctrl`+`b` \| _Right-click²_  |
  | click on `APP_SWITCH`                  | `Ctrl`+`m`                    |
+ | shake device                           | `Ctrl`+`s`                    |
```

Comment:
For consistency:

> click on `MENU`

Reply:
There are also shortcut descriptions for `scrcpy --help` in `main.c`. Could you also add one for `MENU` below `APP_SWITCH` like here?